### PR TITLE
Replace assert by simpler but equivalent calls.

### PR DESCRIPTION
**Pull request by arturobernalg**

### CODE REVIEW

File: maven-scm-providers/maven-scm-provider-perforce/src/test/java/org/apache/maven/scm/provider/perforce/command/unedit/PerforceUnEditConsumerTest.java [hunk 1]
```diff
@@ -59,7 +59,7 @@ public void testBadParse()
         ConsumerUtils.consumeFile( testFile, consumer );
 
         assertFalse( consumer.isSuccess() );
-        assertTrue( consumer.getOutput().indexOf( ".classpath - file(s) not opened on this client." ) == 0 );
+        assertEquals(0, consumer.getOutput().indexOf(".classpath - file(s) not opened on this client."));
         assertTrue( consumer.getOutput().indexOf( ".project - file(s) not opened on this client." ) > 0 );
```
Comment (michael-o) [inline maven-scm-providers/maven-scm-provider-perforce/src/test/java/org/apache/maven/scm/provider/perforce/command/unedit/PerforceUnEditConsumerTest.java:63]: If this line uses true, why equals then?

File: maven-scm-providers/maven-scm-provider-starteam/src/test/java/org/apache/maven/scm/provider/starteam/command/add/StarteamAddConsumerTest.java [hunk 2]
```diff
@@ -65,7 +65,7 @@ public void testParse()
 
             assertTrue( entry.getPath().startsWith( "./" ) );
 
-            assertTrue( entry.getStatus() == ScmFileStatus.ADDED );
+            assertSame(entry.getStatus(), ScmFileStatus.ADDED);
```
Comment (michael-o) [inline maven-scm-providers/maven-scm-provider-starteam/src/test/java/org/apache/maven/scm/provider/starteam/command/add/StarteamAddConsumerTest.java:68]: Spaces lost

File: maven-scm-providers/maven-scm-provider-starteam/src/test/java/org/apache/maven/scm/provider/starteam/command/changelog/StarteamChangeLogConsumerTest.java [hunk 3]
```diff
@@ -77,8 +77,7 @@ public void testNumberOfModifications()
 
         for ( Iterator<ChangeSet> i = entries.iterator(); i.hasNext(); )
         {
-            assertTrue( "ChangeLogEntry erroneously picked up",
-                        i.next().toString().indexOf( "ChangeLogEntry.java" ) == -1 );
+            assertEquals("ChangeLogEntry erroneously picked up", -1, i.next().toString().indexOf("ChangeLogEntry.java"));
```
Comment (michael-o) [inline maven-scm-providers/maven-scm-provider-starteam/src/test/java/org/apache/maven/scm/provider/starteam/command/changelog/StarteamChangeLogConsumerTest.java:80]: Spaces lost

File: maven-scm-providers/maven-scm-provider-starteam/src/test/java/org/apache/maven/scm/provider/starteam/command/checkin/StarteamCheckInConsumerTest.java [hunk 4]
```diff
@@ -62,7 +62,7 @@ public void testParse()
         {
             assertTrue( entry.getPath().startsWith( "./" ) );
 
-            assertTrue( entry.getStatus() == ScmFileStatus.CHECKED_OUT );
+            assertSame(entry.getStatus(), ScmFileStatus.CHECKED_OUT);
```
Comment (michael-o) [inline maven-scm-providers/maven-scm-provider-starteam/src/test/java/org/apache/maven/scm/provider/starteam/command/checkin/StarteamCheckInConsumerTest.java:65]: Spaces lost

File: maven-scm-providers/maven-scm-provider-starteam/src/test/java/org/apache/maven/scm/provider/starteam/command/checkout/StarteamCheckOutConsumerTest.java [hunk 5]
```diff
@@ -64,7 +64,7 @@ private void testParse( File basedir )
         {
             assertTrue( entry.getPath().startsWith( "./" ) );
 
-            assertTrue( entry.getStatus() == ScmFileStatus.CHECKED_OUT );
+            assertSame(entry.getStatus(), ScmFileStatus.CHECKED_OUT);
```
Comment (michael-o) [inline maven-scm-providers/maven-scm-provider-starteam/src/test/java/org/apache/maven/scm/provider/starteam/command/checkout/StarteamCheckOutConsumerTest.java:67]: Spaces lost

File: maven-scm-providers/maven-scm-provider-starteam/src/test/java/org/apache/maven/scm/provider/starteam/command/remove/StarteamRemoveConsumerTest.java [hunk 6]
```diff
@@ -62,7 +62,7 @@ public void testParse()
 
             assertTrue( entry.getPath().startsWith( "./" ) );
 
-            assertTrue( entry.getStatus() == ScmFileStatus.DELETED );
+            assertSame(entry.getStatus(), ScmFileStatus.DELETED);
```
Comment (michael-o) [inline maven-scm-providers/maven-scm-provider-starteam/src/test/java/org/apache/maven/scm/provider/starteam/command/remove/StarteamRemoveConsumerTest.java:65]: Spaces lost

File: maven-scm-providers/maven-scm-providers-cvs/maven-scm-provider-cvs-commons/src/test/java/org/apache/maven/scm/provider/cvslib/command/changelog/CvsChangeLogConsumerTest.java [hunk 7]
```diff
@@ -71,8 +71,7 @@ public void testParse()
         for ( Iterator<ChangeSet> i = entries.iterator(); i.hasNext(); )
         {
             entry = i.next();
-            assertTrue( "ChangeLogEntry erroneously picked up",
-                        entry.toString().indexOf( "ChangeLogEntry.java" ) == -1 );
+            assertEquals("ChangeLogEntry erroneously picked up", -1, entry.toString().indexOf("ChangeLogEntry.java"));
```
Comment (michael-o) [inline maven-scm-providers/maven-scm-providers-cvs/maven-scm-provider-cvs-commons/src/test/java/org/apache/maven/scm/provider/cvslib/command/changelog/CvsChangeLogConsumerTest.java:74]: Spaces lost

File: maven-scm-test/src/main/java/org/apache/maven/scm/tck/command/blame/BlameCommandTckTest.java [hunk 8]
```diff
@@ -88,7 +88,7 @@ public void testBlameCommand()
         assertNotNull( "Expected not null revision", line.getRevision() );
         assertNotNull( "Expected not null date", line.getDate() );
 
-        assertTrue( "Expected another revision", !initialRevision.equals( line.getRevision() ) );
+        assertFalse("Expected another revision", initialRevision.equals(line.getRevision()));
```
Comment (michael-o) [inline maven-scm-test/src/main/java/org/apache/maven/scm/tck/command/blame/BlameCommandTckTest.java:91]: Spaces lost

File: maven-scm-test/src/main/java/org/apache/maven/scm/tck/command/mkdir/MkdirCommandTckTest.java [hunk 9]
```diff
@@ -45,7 +45,7 @@ public void testMkdirCommandMkdirLocal()
 
         assertNotNull( result.getCreatedDirs() );
 
-        assertTrue( "Directory should have been added.", 1 == result.getCreatedDirs().size() );
+        assertEquals("Directory should have been added.", 1, result.getCreatedDirs().size());
```
Comment (michael-o) [inline maven-scm-test/src/main/java/org/apache/maven/scm/tck/command/mkdir/MkdirCommandTckTest.java:48]: Spaces lost

**Review by michael-o — CHANGES_REQUESTED** [review]

**Comment by arturobernalg** [comment]
Hi @michael-o 
Fix it all the remark.
TY

**Review by michael-o — CHANGES_REQUESTED** [review]
There are still spaces lost. Please fix.

**Comment by arturobernalg** [comment]
> There are still spaces lost. Please fix.

HI @michael-o 
I think there all done.
TY

File: maven-scm-providers/maven-scm-providers-cvs/maven-scm-provider-cvs-commons/src/test/java/org/apache/maven/scm/provider/cvslib/command/changelog/CvsChangeLogConsumerTest.java [hunk 10]
```diff
@@ -71,8 +71,8 @@ public void testParse()
         for ( Iterator<ChangeSet> i = entries.iterator(); i.hasNext(); )
         {
             entry = i.next();
-            assertTrue( "ChangeLogEntry erroneously picked up",
-                        entry.toString().indexOf( "ChangeLogEntry.java" ) == -1 );
+            assertEquals( "ChangeLogEntry erroneously picked up",
```
Comment (michael-o) [inline maven-scm-providers/maven-scm-providers-cvs/maven-scm-provider-cvs-commons/src/test/java/org/apache/maven/scm/provider/cvslib/command/changelog/CvsChangeLogConsumerTest.java:74]: The purpose of this test is contains, and not the -1 whi h is a technical detail.

File: maven-scm-providers/maven-scm-provider-starteam/src/test/java/org/apache/maven/scm/provider/starteam/command/changelog/StarteamChangeLogConsumerTest.java [hunk 11]
```diff
@@ -77,8 +77,8 @@ public void testNumberOfModifications()
 
         for ( Iterator<ChangeSet> i = entries.iterator(); i.hasNext(); )
         {
-            assertTrue( "ChangeLogEntry erroneously picked up",
-                        i.next().toString().indexOf( "ChangeLogEntry.java" ) == -1 );
+            assertEquals( "ChangeLogEntry erroneously picked up",
+                    -1, i.next().toString().indexOf("ChangeLogEntry.java") );
```
Comment (michael-o) [inline maven-scm-providers/maven-scm-provider-starteam/src/test/java/org/apache/maven/scm/provider/starteam/command/changelog/StarteamChangeLogConsumerTest.java:81]: Eame here

File: maven-scm-providers/maven-scm-providers-cvs/maven-scm-provider-cvs-commons/src/test/java/org/apache/maven/scm/provider/cvslib/command/changelog/CvsChangeLogConsumerTest.java (hunk 10 above)
Comment (arturobernalg) [reply maven-scm-providers/maven-scm-providers-cvs/maven-scm-provider-cvs-commons/src/test/java/org/apache/maven/scm/provider/cvslib/command/changelog/CvsChangeLogConsumerTest.java]: Revert it

File: maven-scm-providers/maven-scm-provider-starteam/src/test/java/org/apache/maven/scm/provider/starteam/command/changelog/StarteamChangeLogConsumerTest.java (hunk 11 above)
Comment (arturobernalg) [reply maven-scm-providers/maven-scm-provider-starteam/src/test/java/org/apache/maven/scm/provider/starteam/command/changelog/StarteamChangeLogConsumerTest.java]: Revert it

**Review by michael-o — APPROVED** [review]

**Pull request closed without merging**